bundles/org.openhab.binding.weathercompany/README.md

```diff
@@ -0,0 +1,349 @@
+# The Weather Company Binding
+
+Provides 5-day weather forecast and *Personal Weather Station* (PWS) current
+observations from **The Weather Company**.
+This service is available only for PWS users who upload their PWS
+weather data to WeatherUnderground.
+
+## Supported Things
+
+The following thing types are supported:
+
+- Account (bridge) thing for maintaining API key for accessing Weather Company API,
+- Forecast thing for handling the Weather Company 5-day weather forecast, and
+- Observations thing for handling the Personal Weather Station (PWS) weather observations.
+
+## Discovery
+
+Once an Account thing is configured with a valid API key, the binding will auto-discover 
+a *Local Weather Forecast* thing if the location (i.e. latitude and longitude)
+and locale are set in the openHAB configuration.
+
+## Thing Configuration
+
+### Account Thing
```

Comment:
I'd leave out the "Thing" from these titles.
```suggestion
### Account
```

Reply:
Ok